egov/egov-egi/src/main/java/org/egov/infra/workflow/multitenant/model/Attribute.java

```diff
@@ -0,0 +1,63 @@
+package org.egov.infra.workflow.multitenant.model;
+
```

Comment:
**multitenant** can't be a package for workflow
package name **model** should not be used, use either **entity** or **dto**